README.md

```diff
@@ -39,3 +39,18 @@ RUSTFLAGS="--cfg tokio_unstable" CARGO_TARGET_DIR=target/tokio_unstable cargo bu
 ```
 
 Install `tokio-console`, run the server, and run `tokio-console http://127.0.0.1:6669` to connect to it and monitor tasks.
+
+## OpenTelemetry with Jaeger
+
+There is experimental support for exporting application traces to Jaeger. When enabled, tracing spans will be mapped to OpenTelemetry spans, and trace data will be published to a local Jaeger agent via UDP. Enable this feature by compiling with the `jaeger` feature enabled, and add the following to the configuration file.
```

Comment:
Maybe a link here to a website explaining what Jaeger is and how it relates to OpenTelemetry?